commit 8449a9d78509ef68f622e5db31e9a314788ad0f0 (refs/changes/73/3373/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T12:29:15-08:00 (1 year, 8 months ago)
    
    TRITON-150 some node-sdc-clients tests are creating VMs without a cpu_cap

diff --git a/test/cnapi.test.js b/test/cnapi.test.js
index fa89ab6..de24a30 100644
--- a/test/cnapi.test.js
+++ b/test/cnapi.test.js
@@ -221,7 +221,8 @@ test('cnapi', function (tt) {
                     }
                 ]
             },
-            ram: 128
+            ram: 128,
+            cpu_cap: 100
         };
 
         cnapi.createVm(SERVER, opts, function (err, task) {
diff --git a/test/vmapi.test.js b/test/vmapi.test.js
index 31ad342..c3365fd 100644
--- a/test/vmapi.test.js
+++ b/test/vmapi.test.js
@@ -409,6 +409,7 @@ test('vmapi', function (tt) {
             networks: [ ADMIN_NETWORK ],
             brand: 'joyent-minimal',
             ram: 64,
+            cpu_cap: 100,
             server_uuid: HEADNODE.uuid,
             origin: 'sdc-clients-test',
             owner_uuid: CUSTOMER,
